agent/drive_handler.go

```diff
@@ -146,3 +164,87 @@ func isStubDrive(d drive) bool {
 
 	return internal.IsStubDrive(f)
 }
+
+func (dh driveHandler) MountDrive(ctx context.Context, req *drivemount.MountDriveRequest) (*empty.Empty, error) {
+	logger := log.G(ctx).WithField("MountDriveRequest", req.String())
+	logger.Debug()
+
+	driveID := strings.TrimSpace(req.DriveID)
```

Comment:
Can't we just use req.DriveID as is?

Reply:
Yeah I think so, this was just copied from the previous implementation, but unless I'm missing something `TrimSpace` shouldn't be needed, will update